main.go

```diff
@@ -76,7 +81,18 @@ func main() {
 	config.QPS = float32(*clientGoQPS)
 	config.Burst = *clientGoBurst
 
-	stopServer, err := server.Start(config, *discoveryInterval, *informerRelist, *workers)
+	options := options.Options{
```

Comment:
I felt placing Options within specific packages will make things more cohesive. 
However, we shall take a look at next PR.